cookbooks/aws-parallelcluster-slurm/recipes/config_head_node.rb

```diff
@@ -192,7 +192,7 @@
     mode '0755'
   end
 
-  cookbook_file "#{node['cluster']['slurm']['install_dir']}/etc/pcluster/prolog.d/pcluster_prolog" do
+  cookbook_file "#{node['cluster']['slurm']['install_dir']}/etc/pcluster/prolog.d/01pcluster_prolog" do
```

Comment:
I'd rename from `01pcluster_prolog` to `01_pcluster_prolog` to be aligned to other config (e.g.: 
`/etc/security/limits.d/00_all_limits.conf`)

Reply:
I'm going to use same convention used in `/etc/udev/rules.d/`, which is using a dash "-"